src/overlays/actors/ovl_En_Bee/z_en_bee.h

```diff
@@ -14,14 +14,14 @@ typedef struct EnBee {
     /* 0x188 */ Vec3s morphTable[OBJECT_BEE_LIMB_MAX];
     /* 0x1C4 */ Vec3s jointTable[OBJECT_BEE_LIMB_MAX];
     /* 0x200 */ EnBeeActionFunc actionFunc;
-    /* 0x204 */ s16 unk_204;
-    /* 0x206 */ s16 unk_206;
-    /* 0x208 */ s16 unk_208;
-    /* 0x20C */ s32 unk_20C;
-    /* 0x210 */ s32 unk_210; 
-    /* 0x214 */ s32 unk_214;
-    /* 0x218 */ s32 unk_218;
-    /* 0x21C */ Vec3f unk_21C[2];
+    /* 0x204 */ s16 attackDelayTimer;
+    /* 0x206 */ s16 sfxHitTimer;
+    /* 0x208 */ s16 isHostile;  // unused
```

Comment:
```suggestion
    /* 0x208 */ s16 isHostile;  // set and not used
```
We like to be a bit more precise since there are plenty of actors that have completely unused struct elements (see a few elements further down in this very struct!).